compiler/optimizer/DeadTreesElimination.cpp

```diff
@@ -306,7 +306,7 @@ static bool isSafeToReplaceNode(TR::Node *currentNode, TR::TreeTop *curTreeTop,
        *    => xload/xloadi a.volatileField
        *    ...
        */
-      //if (mayBeVolatileReference && !canMoveIfVolatile)
+      //if (mayBeNonTransparentReference && !canMoveIfVolatile)
```

Comment:
Comments that appear just above here talk about restrictions on "swinging down" volatile.  Those comments will need to be updated.

Reply:
fixed in https://github.com/eclipse-omr/omr/commit/e6464b49a044099b3a6fd2f1065cfd4df26660a7